### DISCUSSION TITLE
torch.cuda.is_available() error

### DISCUSSION THREAD

**Discussion by ankitmazumder** [mrdbourke/pytorch-deep-learning#305] [Q&A] [answered]
# Setup device agnostic code
device = "cuda" if torch.cuda.is_availble() else "cpu"
device

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-8-53dbd9c29cc4>](https://localhost:8080/#) in <module>
      1 # Setup device agnostic code
----> 2 device = "cuda" if torch.cuda.is_availble() else "cpu"
      3 device

AttributeError: module 'torch.cuda' has no attribute 'is_availble'

How to resolve this error

**Comment by nazarPuriy** [comment, marked as answer]
You misspelled available. 

Try:

```python
import torch
device = "cuda" if torch.cuda.is_available() else "cpu"
```
